Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/DataGrid/DataGrid.xaml

```diff
@@ -477,7 +479,9 @@
                             Content="{TemplateBinding Content}"
                             ContentTemplate="{TemplateBinding ContentTemplate}"
                             HorizontalAlignment="{TemplateBinding HorizontalContentAlignment}"
-                            VerticalAlignment="{TemplateBinding VerticalContentAlignment}"/>
+                            VerticalAlignment="{TemplateBinding VerticalContentAlignment}"
+                            Background="{TemplateBinding Background}" 
+                            Foreground="{TemplateBinding Foreground}"/>
```

Comment:
I am concerned about how this interferes with the row-level Foreground handling done by:
- DataGridCheckBoxColumn.ConfigureCheckBox(CheckBox checkBox, Brush computedRowForeground)
- DataGridCheckBoxColumn.RefreshForeground(FrameworkElement element, Brush computedRowForeground)
- DataGridComboBoxColumn.RefreshForeground(FrameworkElement element, Brush computedRowForeground)
- DataGridTextColumn,RefreshForeground(FrameworkElement element, Brush computedRowForeground)
- DataGrid.RowForeground
- DataGrid.AlternatingRowForeground
The various column types do not do the same processing for the background.
Have you thought about this?   Is the cell-level Foreground being overridden by the row-level Foreground?   Thank you Dean.

Reply:
@deanchalk have you tested these scenarios called out by @RBrid?

Reply:
@deanchalk is this something you still wanted to address?